assets/css/_garage_filter.scss

```diff
@@ -1,37 +1,74 @@
 .m-garage-filter {
   @include font-small;
-  color: #6e717d;
+  font-weight: 400;
+  color: #525560;
```

Comment:
**suggestion (non-blocking):** This value is used in several places. It might be worthwhile turning it into a variable.